### ISSUE TITLE
PureOS setup page: Please warn about incredibly broken and outdated apps

### ISSUE THREAD

**Issue by secretmango** [flatpak/flatpak.github.io#636]
The page basically says "PureOS has it preinstalled, just add Flathub and you're done!" but this is not the case.

At least in KDE Discover, PureOS was the chosen repo BEFORE flathub, at least when added afterwards (other remotes where below that so idk the reason)

And the PureOS remote is bad. The apps are very old, many many duplicates with Flathub, and I got 2 out of 2 tried apps that where just broken, didnt even launch or crash. 

Please warn about this, "disable the PureOS repository if you want to only want to use Flathub". Yes that is kinda nasty to say, but their apps are not ready / outdated prerelease software.

Couldnt find the site so will just yell it out here, otherwise would have done a PR, sorry

**Comment by secretmango** [comment]
https://flathub.org/setup/PureOS

**Comment by AsciiWolf** [comment]
This should ideally be discussed with Purism and their developers/package maintainers. Adding things like "disable the [replace with a distro name] repository" to our setup pages is not a good idea.

**Comment by secretmango** [comment]
Yes agree. But somehow it should be stressed, that the PureOS remote is somehow higher privileged than Flathub, which practically doesnt "enable Flathub" for at least the apps PureOS supports.

**Comment by AsciiWolf** [comment]
Distro's own remotes should have higher priority than Flathub. This is correct. Broken and outdated software in distro's remotes is not. This however needs to be talked with PureOS maintainers. Just disabling their remote does not solve anything.

**Comment by secretmango** [comment]
Dont want to be picky, [but their repos dont sound very alive](https://github.com/purism).

Having a Github profile or mainly archived repos is not thaat promising, that they will actually fix all those apps.

For now, they ship many of Flathubs apps, but old and broken versions of them. I dont know why the PureOS remote is above Flathub, I am on Fedora Kinoite and added it to test it. 

It is not the best solution, but it should be mentioned 

1. in what state the shipped apps are
2. that the remote dominates over Flathub, and users may install unmaintained, outdated and broken apps

Edit: Even though nowhere stated, I guess [This is their own Gitlab instance](https://source.puri.sm/public) which they use instead of Github
